src/main/webapp/js/diagramly/BackendFile.js

```diff
@@ -310,7 +178,45 @@
   */
  BackendFile.prototype.open = function()
  {
-     this.ui.setFileData(this.getData());
-     this.installListeners();
+    this.getFromAPI((resp) => {
+        this.setData(resp);
+        this.ui.setFileData(resp);
+        this.installListeners();
+        DrawioFile.SYNC = 'auto';
+        this.startSync();
+    }, (e) => {
+        this.ui.handleError(e, mxResources.get('errorLoadingFile'));
+    });
+ };
+
+ BackendFile.prototype.getFromAPI = function(success, error)
+ {
+    const opts = {
+        headers: {
+            'Authorization': 'Bearer ' + this.token,
+            'Content-Type': 'application/json'
+        }
+    };
+
+    axios.get(this.endpoint, opts).then((resp) => success(resp.data.xml), error);
+ };
+ 
+ BackendFile.prototype.postToAPI = function(content, success, error)
+ {
+    const opts = {
+        headers: {
+            'Authorization': 'Bearer ' + this.token,
+            'Content-Type': 'application/json'
+        }
+    };
+
+    const idString = this.endpoint.subsring(this.endpoint.lastIndexOf('/') + 1);
```

Comment:
Bement a patch endpoint változtatás, úgyhogy most már nem kell az objektumban is megadni az ID-t.